api/db/migrations/20220411134827_add-column-source-to-complementary-certification-results.js

```diff
@@ -0,0 +1,14 @@
+const TABLE_NAME = 'complementary-certification-course-results';
+const COLUMN_NAME = 'source';
+
+exports.up = async function (knex) {
+  await knex.schema.table(TABLE_NAME, function (table) {
+    table.string(COLUMN_NAME).defaultTo('PIX');
+  });
```

Comment:
await knex.raw(
    'ALTER TABLE ? ADD CONSTRAINT "complementary-certification-course-results_source_check" CHECK ( ? IN ( \'PIX\', \'EXTERNAL\'), [TABLE_NAME, COLUMN_NAME])'
  );

Reply:
```
  await knex.raw(`
    ALTER TABLE "complementary-certification-course-results" ADD CONSTRAINT "complementary-certification-course-results_source_check"
      CHECK ( "source" IN ( 'PIX', 'EXTERNAL'))
  `);
```